src/vsr.zig

```diff
@@ -1374,11 +1375,11 @@ pub const Checkpoint = struct {
         const result = op: {
             if (checkpoint == 0) {
                 // First wrap: op_checkpoint_next = 8-2-1 = 5
-                break :op constants.journal_slot_count - constants.lsm_batch_multiple - 1;
+                break :op constants.vsr_checkpoint_interval - 1;
             } else {
                 // Second wrap: op_checkpoint_next = 5+8-2 = 11
                 // Third wrap: op_checkpoint_next = 11+8-2 = 17
```

Comment:
Can you update these "First/Second/Third wrap" notes to include a pipeline?
Also the diagram above `pub const Checkpoint`